swarm/network/stream/messages.go

```diff
@@ -96,26 +104,29 @@ func (p *Peer) handleSubscribeMsg(ctx context.Context, req *SubscribeMsg) (err e
 			// exchange between peers over p2p. Instead, error will be returned
 			// only if there is one from sending subscribe error message.
 			err = p.Send(context.TODO(), SubscribeErrorMsg{
-				Error: err.Error(),
+				Server: true,
```

Comment:
In this case this is true, it is the client who sends a `SubscribeMsg` (when running `handleRequestSubscription`) to the server, which itself runs `handleSubscribeMsg`, thus if there is an error here, this field is indeed true.